packages/flutter_tools/test/general.shard/android/android_device_test.dart

```diff
@@ -124,6 +82,20 @@ void main() {
         )).thenAnswer((_) async {
           return ProcessResult(0, 0, '[ro.build.version.sdk]: [24]', '');
         });
+        when(mockProcessManager.run(
```

Comment:
This is fine, but the `FakeProcessManager` in `fake_process_manager.dart` is pretty nice to use.

Reply:
ack :)